tests/unit_test.py

```diff
@@ -975,41 +977,53 @@ def get_ray_obj(group, version, namespace, plural, cls=None):
                                         "f:serviceType": {},
                                         "f:template": {
                                             ".": {},
-                                            "f:spec": {".": {}, "f:containers": {}},
+                                            "f:spec": {
+                                                ".": {},
+                                                "f:affinity": {
+                                                    ".": {},
+                                                    "f:nodeAffinity": {
+                                                        ".": {},
+                                                        "f:requiredDuringSchedulingIgnoredDuringExecution": {},
+                                                    },
+                                                },
+                                                "f:imagePullSecrets": {},
+                                                "f:volumes": {},
+                                            },
                                         },
                                     },
                                     "f:rayVersion": {},
                                     "f:workerGroupSpecs": {},
                                 },
                             },
-                            "manager": "mcad-controller",
+                            "manager": "codeflare-operator",
                             "operation": "Update",
-                            "time": "2023-02-22T16:26:07Z",
+                            "time": "2024-03-05T09:55:37Z",
                         },
                         {
-                            "apiVersion": "ray.io/v1",
```

Comment:
Should be the most up to date here too I think, so v1 and not v1alpha1. I know its in unit tests so a nitpick.